src/main/kotlin/no/nav/syfo/infrastructure/database/repository/VedtakRepository.kt

```diff
@@ -283,7 +283,7 @@ class VedtakRepository(private val database: DatabaseInterface) : IVedtakReposit
 
         private const val GET_UNPUBLISHED_INFOTRYGD =
             """
-                SELECT * FROM VEDTAK WHERE published_infotrygd_at IS NULL
+                SELECT * FROM VEDTAK WHERE published_infotrygd_at IS NULL AND created_at < now() - interval '1 minutes'
```

Comment:
Lagt inn en ekstra sjekk på `created_at` for å unngå at cronjobben går i beina på publiseringen som skjer når vedtaket lages.